<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBudgetView</class>
 <widget class="QWidget" name="KBudgetView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="groupBox1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Your budgets</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="m_newButton">
           <property name="toolTip">
            <string>Creates a new budget</string>
           </property>
           <property name="whatsThis">
            <string>Use this to create a new empty budget.</string>
           </property>
           <property name="text">
            <string comment="New budget">New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_renameButton">
           <property name="toolTip">
            <string>Rename the current selected budget.</string>
           </property>
           <property name="whatsThis">
            <string>Use this to start renaming the selected budget.</string>
           </property>
           <property name="text">
            <string>Ren</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_deleteButton">
           <property name="toolTip">
            <string>Delete the current selected budget</string>
           </property>
           <property name="whatsThis">
            <string>Use this to delete the selected budget.</string>
           </property>
           <property name="text">
            <string>Del</string>
           </property>
           <property name="shortcut">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="m_budgetList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string comment="@title header of the budget name column">Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="KMyMoneyAccountTreeView" name="m_accountTree"/>
       </item>
       <item>
        <widget class="QLabel" name="m_balanceLabel">
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="m_updateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Accepts the entered values and stores the budget.</string>
           </property>
           <property name="whatsThis">
            <string>Use this to store the modified data.</string>
           </property>
           <property name="text">
            <string>Upd</string>
           </property>
           <property name="shortcut">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_resetButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Revert budget to last saved state.</string>
           </property>
           <property name="whatsThis">
            <string>Use this to discard the modified data.</string>
           </property>
           <property name="text">
            <string>Res</string>
           </property>
           <property name="shortcut">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_collapseButton"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_expandButton"/>
         </item>
         <item>
          <widget class="QCheckBox" name="m_hideUnusedButton">
           <property name="text">
            <string>Hide unused budget categories</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="m_searchWidget">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="m_assignmentBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Assignments</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel3">
              <property name="text">
               <string>Account</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KLineEdit" name="m_leAccounts">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string comment="Budget total">Total</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="AmountEdit" name="m_accountTotal">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="calculatorButtonVisibility" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resetButtonVisibility" stdset="0">
               <bool>false</bool>
              </property>
              <property name="readOnly" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cbBudgetSubaccounts">
            <property name="text">
             <string>Include subaccounts</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="KBudgetValues" name="m_budgetValue" native="true"/>
            </item>
            <item>
             <spacer name="spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountTreeView</class>
   <extends>QTreeView</extends>
   <header>kmymoneyaccounttreeview.h</header>
  </customwidget>
  <customwidget>                                   
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>                   
   <header location="global">amountedit.h</header>
  </customwidget>
  <customwidget>
   <class>KBudgetValues</class>
   <extends>QWidget</extends>
   <header>kbudgetvalues.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_collapseButton</sender>
   <signal>clicked()</signal>
   <receiver>m_accountTree</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_expandButton</sender>
   <signal>clicked()</signal>
   <receiver>m_accountTree</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
